stripe.go

```diff
@@ -72,6 +73,8 @@ func (a *AppInfo) formatUserAgent() string {
 // This interface exists to enable mocking for during testing if needed.
 type Backend interface {
 	Call(method, path, key string, body *form.Values, params *Params, v interface{}) error
+	Call2(method, path, key string, params ParamsContainer, v interface{}) error
+	CallRaw(method, path, key string, body *form.Values, params *Params, v interface{}) error
```

Comment:
`Call2` will become `Call`. `CallRaw` needs to stick around for now so that we can still implement the iterator closures in our list functions (we may be able to simplify those in the future too, but I haven't looked into it yet).